<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlight</class>
 <widget class="QDialog" name="AddFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="AddFight_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>添加航班信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CityStart_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CityStart_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>出发城市</string>
   </property>
  </widget>
  <widget class="QLabel" name="CityEnd_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>到达城市</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CityEnd_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="FightNumber_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="FightNumber_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>航班号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlaneNumber_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PlaneNumber_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>飞机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeStart_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>预计起飞时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeEnd_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>预计到达时间</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="TimeStart">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="TimeEnd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Price_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>票价</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Price_Line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Discount_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>折扣</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Discout_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PeopleNumber_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>乘员定额</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PeopleNumber_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TicketLeft_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>余票量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TicketLeft_Line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
